src/HyperLogLog.net/HyperLogLog.net/HyperLogLog.cs

```diff
@@ -80,11 +92,14 @@ public int GetCount()
                 else
                     dv = (int)(M * Math.Log((double)M / v));
             }
-            else if (dvEstimate <= (Pow2_32 * 1 / 30))
+            else if (dvEstimate <= (Pow2_32 * 1 / 30))  // 143,165,576
                 dv = (int)dvEstimate;
             else
             {
-                dv = (int)(Math.Pow(-2, 32) * Math.Log(1 - dvEstimate / Pow2_32));
+                if (dvEstimate <= 1E9)
+                    dv = (int)(-Pow2_32 * Math.Log(1 - dvEstimate / Pow2_32));
```

Comment:
My math isn't very good, but according to Bing, `Math.Pow(-2, 32)` equals `Pow2_32`, and not `-Pow2_32`: http://www.bing.com/search?q=(-2)%5E32